### DISCUSSION TITLE
New WPF project from scratch.

### DISCUSSION THREAD

**Discussion by doraemon0044** [Live-Charts/LiveCharts2#105] [General]
Hi everyone, I would like to know how to build a project from scratch because l confuse making use of Viewmodel  in XAML file.

**Comment by CodeWithMichal** [comment]
Hi, I think first you should learn about WPF itself since it seems that you are lacking some basic knowledge without which it's really difficult to talk about LiveCharts.
Still, I would love to help you but please rephrase your request. Right know I am not sure what exactly you want to know
